srcpkgs/blender/template

```diff
@@ -54,8 +54,8 @@ case "$XBPS_TARGET_MACHINE" in
 esac
 
 case "$XBPS_TARGET_MACHINE" in
-x86_64*)
-	# Enable Intel Open Image Denoise on x86_64
+x86_64)
+	# Intel(R) Open Image Denoise supports glibc 64-bit platforms only
```

Comment:
yet srcpkgs/openimagedenoise-devel/template has `archs="x86_64*"` set (not pinning it to glibc), can you clarify and add a fix to either one of them?

Reply:
https://github.com/void-linux/void-packages/pull/22562 here i did this.